docs/how-to/h-enable-monitoring.md

```diff
@@ -0,0 +1,99 @@
+Creating and listing backups requires that you:
+* [Have a Charmed MySQL deployed](https://discourse.charmhub.io/t/charmed-mysql-tutorial-managing-units/TODO)
+* [Deploy `cos-lite` bundle in a Kubernetes environment](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s)
+
+Switch to COS K8s environment and offer COS interfaces to be cross-model related with Charmed MySQL VM model:
+```shell
+# Switch to Kubernetes controller, for the cos model.
+juju switch <k8s_controller>:<cos_model_name>
+
+juju offer grafana.grafana-dashboards grafana-dashboards
+juju offer loki:logging loki-logging
+juju offer prometheus:receive-remote-write prometheus-receive-remote-write
+```
+
+Switch to Charmed MySQL VM model, find offers and relate with them:
+```shell
+# We are on the Kubernetes controller, for the cos model. Switch to mysql model
+juju switch <machine_controller_name>:<mysql_model_name>
+
+juju find-offers <k8s_controller>:
+```
+
+A similar output should appear, if `k8s` is the k8s controller name and `cos` the model where `cos-lite` has been deployed:
+```shell
+Store      URL                                        Access  Interfaces
+k8s        admin/cos.grafana-dashboards               admin   grafana_dashboard:grafana-dashboard
+k8s        admin/cos.loki-logging                     admin   loki_push_api:logging
+k8s        admin/cos.prometheus-receive-remote-write  admin   prometheus-receive-remote-write:receive-remote-write
+...
+```
+
+Consume offers to be reachable in the current model:
+```shell
+juju consume k8s:admin/cos.prometheus-receive-remote-write
+juju consume k8s:admin/cos.loki-logging
+juju consume k8s:admin/cos.grafana-dashboards
+```
+
```

Comment:
we can maybe show that the consumed endpoints should appear in the SAAS section for the `juju status` command